core/src/com/fourinachamber/fortyfive/game/SaveState.kt

```diff
@@ -94,6 +94,16 @@ object SaveState {
         savefileDirty = true
     }
 
+    /**
+     * the current money of the player
+     */
+    var playerMoney: Int by templateParam("stat.playerMoney", 0) {
```

Comment:
doesn't matter much, but the templateParam should use a different prefix than stat, which is used for statistics

Reply:
but YOU used it for the playerLifes as well, even though it is saved "different" from the stats